src/components/FileTree/index.tsx

```diff
@@ -38,7 +38,7 @@ export type DefaultProps = {
 };
 
 type PropsFromState = {
-  tree: DirectoryNode | void;
+  treeNodes: DirectoryNode | void;
```

Comment:
It would be helpful to make this `treeNodes: FileTree['nodes']` so they remain in sync.